app/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/javalin/DomainsRegistrationTest.groovy

```diff
@@ -26,6 +28,7 @@ class DomainsRegistrationTest extends Specification {
         javalinContext.body() >> bodyString
         javalinContext.url() >> urlString
         javalinContext.headerMap() >> headerMap
+        ApplicationContext.register(TiLogger.class, TiLogger.getLogger()) // Needed since DomainRegistrationTest resets it
 
```

Comment:
Maybe it can go in a def setup(){} / def setupSpec(){} method if more test cases need it.

Reply:
I would be OK to changing this to `TestApplicationContext.register(...)` and in the constructor of the `TestApplicationContext`, we always register the logger.  Also, it should be registered under `Logger.class`, not `Slf4jLogger.class`.